src/patternfly/components/DataToolbar/data-toolbar-item-overflow-menu.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-data-toolbar__item pf-m-overflow-menu{{#if data-toolbar-overflow-menu--modifier}} {{data-toolbar-overflow-menu--modifier}}{{/if}}"
+  {{#if data-toolbar-overflow-menu--attribute}}
```

Comment:
same here, should you use `data-toolbar-item-overflow-menu--[attribute/modifier]`?